modules/serverless-creating-subscriptions-ODC.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-creating-subscriptions-ODC_{context}"]
+= Connecting a channel to a sink using subscriptions in {product-title} web console
+
+You can create a subscription to connect a service or other event sink to a channel through the {product-title} web console.
+
+.Prerequisites
+* To create a subscription through the web console, you must already have created a Knative service that can be used as an event sink.
```

Comment:
Suggestion: If you change the prerequisite a bit like this: ...you must already have created a Knative service that can be used as an event sink...in the same namespace.
You can then add a simple procedure to drag the arrow to the service.

Reply:
They don't _have to_ be in the same namespace though afaik, so I wouldn't put this as a prerequisite :slightly_smiling_face: